[Infra] Fix regex library warnings in skip_pod_lint.py

emmanuel-ferdman opened the pull request:
<!--
  Thank you for submitting a pull request!

  We appreciate the time and effort you have invested in making these changes. Please ensure that you provide enough information to allow others to review your pull request.

  Upon submission, your pull request will be automatically assigned with reviewers.

  If you want to learn more about contributing to this project, please visit: https://github.com/lynx-family/lynx/blob/develop/CONTRIBUTING.md.
-->

## Summary
This small PR resolves the regex library warnings:
```python
/tmp/lynx/tools/ios_tools/skip_pod_lint.py:23: DeprecationWarning: 'count' is passed as positional argument
```

## Checklist

<!--- Check and mark with an "x" -->

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/lynx-family/lynx?pullRequest=1238) <br/>All committers have signed the CLA.

coolkiid commented:
@gongfeng98 PTAL

coolkiid commented:
@emmanuel-ferdman Thanks for your contribution <3

Please modify your PR and commit title as the instructions in the [contributing doc](https://github.com/lynx-family/lynx/blob/develop/CONTRIBUTING.md#pull-requests). [Infra] might be a better label :)

emmanuel-ferdman committed:
[Infra] Fix regex library warnings in skip_pod_lint.py

Summary of change:
The 'count' parameter in re.sub() was being passed as a positional argument,
which is deprecated in Python 3.7 and will be removed in future versions.
This change converts it to a keyword argument to suppress the DeprecationWarning
and ensure future compatibility.

Signed-off-by: Emmanuel Ferdman <emmanuelferdman@gmail.com>

emmanuel-ferdman commented:
@coolkiid fixed, hope it's ok now. thanks for the review!

gongfeng98 commented:
/land

jianliang00 commented:
🛬 Landing process started! >>567.0

jianliang00 commented:
🛬 Landing process started! >>568.0
